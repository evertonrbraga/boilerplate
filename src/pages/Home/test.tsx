import { render, screen } from 'project-testing-library'
import { Home } from '.'

describe('<Home />', () => {
  it('should snapshot the Home page', () => {
    const { container } = render(<Home />)
    expect(container.firstChild).toMatchSnapshot()
  })

  it('should if the title is rendering correctly', () => {
    render(<Home />)
    const title = screen.getByText('Home')
    expect(title).toBeInTheDocument()
  })
})
