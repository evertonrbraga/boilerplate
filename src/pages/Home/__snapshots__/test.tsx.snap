// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should snapshot the Home page 1`] = `
.c0 {
  background: #323238;
}

<main
  class="c0"
>
  <h1>
    Home
  </h1>
</main>
`;
